src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -186,6 +186,13 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+       public override void Reset()
+        {
+            this.PRTitle = string.Empty;
+            this.Description = string.Empty;
+            TargetBranch = Branches.FirstOrDefault(b => b.Name == "master");
```

Comment:
The `TargetBranch` default value is not hardcoded to master anymore, it's whatever it gets set to in https://github.com/github/VisualStudio/blob/35d1a987f7750d3360e2a6718a823fbf635f66c6/src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs#L69 from the value in the `GitHubRepository` property.